setup.py

```diff
@@ -15,7 +15,8 @@
                      "easydict",
                      "lxml",
                      "requests",
-                     "gpxpy"
+                     "gpxpy",
+                     "dataclasses"
```

Comment:
I wonder if there is a way of preventing installation of this package on Python >= 3.7 – there, dataclasses are in the standard library.

Reply:
Aah, ok. I was having the issue because I am still on 3.6.9...

Reply:
> 
> 
> I wonder if there is a way of preventing installation of this package on Python >= 3.7 – there, dataclasses are in the standard library.

Ah, very good idea. I hadn't thougth about it. It is accutaly possible. -> https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-platform-specific-dependencies